### PR TITLE
remove nix-index-database from rpi

### PR DESCRIPTION
**Pull request by DerDennisOP**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -206,9 +207,10 @@
               }) users) else [ ])
               ++ lib.optional (system != "x86_64-linux") {
                 nixpkgs.overlays = [
-                  (_self: super: {
-                    pcsclite = patch-pcsclite.legacyPackages.${system}.pcsclite;
-                  })
+                  (_self: super: (builtins.listToAttrs (map (name: {
+                    name = name;
+                    value = inputs.patch-systemd.legacyPackages.${system}.${name};
+                  }) (builtins.attrNames inputs.patch-systemd.legacyPackages.${system}))))
```
Comment (ahuston-0) [inline flake.nix:213]: looks like `patch-pcsclite.url` is no longer getting used in favor of `patch-systemd`... do we still need the first one?

**Review by ahuston-0 — DISMISSED** [review]
approved besides that one thing i asked

File: flake.nix (hunk 1 above)
Comment (DerDennisOP) [reply flake.nix:213]: uhm I'll clean it up

**Review by ahuston-0 — APPROVED** [review]
re-approved

**Pull request merged**